packages/components/src/Slider/Slider.component.js

```diff
@@ -218,6 +218,8 @@ class Slider extends React.Component {
 		captionTextStepNumber: PropTypes.number,
 		min: PropTypes.number.isRequired,
 		max: PropTypes.number.isRequired,
+		rightTrack: PropTypes.bool,
```

Comment:
what do you think about a `trackPosition` (by default set to' Left') ['Right', 'Left', 'None'] instead?

Reply:
Agree, instead of multiplying the props for 1 purpose, let's enable different modes via 1 prop.
I would prefer `props.mode` with different values (`lowerThan`, `between`, `greaterThan`, `equals`). We need to speak the same concept/language as the UX in the guideline

Reply:
Thank you both. I had considered this after and I agree, one prop is cleaner. In my opinion, since this is a visual change only, the styles should not have an implied context and should be generic. On that basis, I would prefer `left, right, none` instead of `lessThan, GreaterThan, equals`.

If we do this, `left` will be the default prop value but it will have no dedicated modifier class unless we want that? If we do, it will be a breaking change. 

For range, this was not in scope for my ticket so the behaviour is not currently modified. There is an issue where you could apply modifiers to a range but I will fix this. If we want to handle the range like `left, right`, we could have `inner, outer` to support an inclusive or exclusive range. The final 'mode' options would be:

```
- none
- left
- right
- inner
- outer
```

Reply:
Can you explain the `since this is a visual change only` ?
What is plained for the behavior (non-visual part) ? Are you planning to return the value or the corresponding range depending on the mode ?
Either way, instead of mapping visual values to the real meaning (left = less than the selected value), I think it's clearer to have directly the meaning. And if there is some behavior change that are planned, we can directly use those values too, no mental mapping.
But if you both agree on the left/right values, no blocker, let's go with that.

Reply:
There will be no change to the functional behaviour of the slider. It will return a value or range as normal which can be interpreted by the app in whatever context is needed. 

Okay, so maybe we can have:
```
- lessThan
- equals
- greaterThan
- inclusive (for range)
- exclusive (for range)
```

Reply:
sure, go for it!